src/EPR.Payment.Facade.Common/Dtos/Response/RegistrationFees/RegistrationFeesResponseDto.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace EPR.Payment.Facade.Common.Dtos.Response.RegistrationFees
+{
+    public class RegistrationFeesResponseDto
+    {
+        public decimal? BaseFee { get; set; } // Nullable to handle scenarios where the fee is not applicable
+        public decimal? SubsidiariesFee { get; set; } // Nullable to handle scenarios where the fee is not applicable
+        public decimal TotalFee { get; set; }
+        public List<FeeBreakdown> FeeBreakdowns { get; set; } = new();
+    }
+
+    public class FeeBreakdown
```

Comment:
Move this to its own file

Reply:
Personally I think in this instance it is fine to have these in the same file. The FeeBreakdown is very closely coupled with the Dto so imo it makes it more readable and easier for other dev to see the relationship.

Reply:
best practice to keep a class in a file, im sue Sonar will raise this, 

if this is the process we are following here then happy to close this comment